lib/components/map/enhanced-stop-marker.js

```diff
@@ -119,7 +120,7 @@ class EnhancedStopMarker extends Component {
       // Instead, we want the tip of the caret at the bottom center of the bubble
       // to be at the stop location.
       // Add some margins so the stop marker (which may be unintentionally offset) remains visible.
-      iconAnchor: [bubblePixels / 2 + 4, bubblePixels + caretVisibleHeight + 8]
+      iconAnchor: [bubblePixels(this.props) / 2 + 4, bubblePixels(this.props) + caretVisibleHeight + 8]
```

Comment:
I think the bubble should be shifted down so that the border where it meets the caret doesn't show.
![image](https://user-images.githubusercontent.com/56846598/141151772-9dc0aaeb-0f7f-40a3-8d83-9e2b2fa0e1d4.png)

Reply:
Fixed in abcff8324907d35f9686a327d8c68a4ca9a283f8